### PR TITLE
data:Himanshuch8055

### PR DESCRIPTION
**Pull request by Himanshuch8055**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3323"></a>

### CODE REVIEW

**Review by Njong392 — DISMISSED** [review]
Looks awesome! Thank you for contributing to LinkFree :tada:

**Comment by Himanshuch8055** [comment]
I'm also thankful. @Njong392 @

File: data/Himanshuch8055.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "bio": "Open source Advocate | Programmer | Content Writer | Developer | Competitive Coder",
+  "tags": ["Open Source", "Content Writer", "Blogging", "Web3", "Developers", "Competitive Coding"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/himanshuch8055" },
+    { "icon": "FaGithub", "url": "https://github.com/Himanshuch8055" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/himanshuchauhan8055/" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/himanshu-chauhan-693404226/" },
+    { "icon": "FaFacebook", "url": "https://www.facebook.com/himanshuchauhan8055" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/channel/UCoBSwNEZRZ1ROny1RNACh7A" },
+    { "icon": "FaDiscord", "url": "https://discord.gg/https://discord.gg/deYQcTGk" },
```
Comment (Panquesito7) [inline data/Himanshuch8055.json:14]: Wrong link here.
```suggestion
    { "icon": "FaDiscord", "url": "https://discord.gg/deYQcTGk" },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Himanshuch8055.json (hunk 1 above)
Comment (Njong392) [reply data/Himanshuch8055.json:14]: > Wrong link here.
> ```suggestion
>     { "icon": "FaDiscord", "url": "https://discord.gg/deYQcTGk" },
> ```
> 

Must've missed that! Thanks 🙌

**Comment by Njong392** [comment]
Hello @Himanshuch8055 Is there any particular reason for closing this before it's merged? I see that you made changes as per the reviewer.

**Comment by Himanshuch8055** [comment]
@Njong392 Actually I'm create new pull request.

**Pull request closed without merging**